WordPress/Classes/ViewRelated/Plugins/CollectionViewContainerRow.swift

```diff
@@ -42,22 +44,41 @@ struct CollectionViewContainerRow<CollectionViewCellType: UICollectionViewCell,
                                                   collectionCellSeletectedBlock: selectionBlock)
     }
 
+    init(title: String,
+         secondaryTitle: String?,
+         action: ImmuTableAction?,
+         noResultsView: WPNoResultsView) {
+        self.title = title
+        self.secondaryTitle = secondaryTitle
+        self.action = action
+        self.noResultsView = noResultsView
+
+        let configurationBlock = { (_: UICollectionViewCell, _: Any) in return }
+        let selectionBlock = { (_: Any) in return }
+
+        helper = CollectionViewHandler(dataSourceItems: [],
+                                       configureCollectionCellBlock: configurationBlock,
+                                       collectionCellSeletectedBlock: selectionBlock)
```

Comment:
You can simplify the empty block syntax:

```swift
        helper = CollectionViewHandler(dataSourceItems: [],
                                       configureCollectionCellBlock: { _, _ in },
                                       collectionCellSeletectedBlock: { _ in })
```

But if this is an option, I'd just make the two blocks optionals and pass `nil`.